server/sonar-web/src/main/js/apps/organizations/components/OrganizationPermissions.tsx

```diff
@@ -17,24 +17,21 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-// @flow
-import React from 'react';
+import * as React from 'react';
 import { connect } from 'react-redux';
 import GlobalPermissionsApp from '../../permissions/global/components/App';
 import { getOrganizationByKey } from '../../../store/rootReducer';
-/*:: import type { Organization } from '../../../store/organizations/duck'; */
+import { Organization } from '../../../app/types';
 
-/*::
-type Props = {
-  organization: Organization
-};
-*/
+interface Props {
+  organization: Organization;
+}
 
-function OrganizationPermissions(props /*: Props */) {
-  return <GlobalPermissionsApp organization={props.organization} />;
+function OrganizationPermissions({ organization }: Props) {
```

Comment:
I wonder if we can avoid using this component by using `OrganizationContainer` instead